CONTRIBUTING.md

```diff
@@ -1,10 +1,12 @@
 # Contributing to DAML
 
-Welcome! This document provides a high-level overview of how to contribute to the development of DAML.
+Welcome! This page gives a high-level overview of how to contribute to the development of DAML.
 
-(For information on how to build, test, and work on the codebase, see the [README](./README.md).)
+There are many ways you can contribute beyond coding. For example, you can report problems, report and clarify issues, and write documentation. If you're completely new to open source development, the [Open Source Guides](https://opensource.guide) is a great place to start.
```

Comment:
What's the difference between "reporting problems" and "reporting issues"?

Reply:
from the original - I thought issues were specifically GH issues, and problems more general, but I agree it's not clear